easybuild/tools/options.py

```diff
@@ -268,6 +269,8 @@ def config_options(self):
                                 "(is passed as list of arguments to create the repository instance). "
                                 "For more info, use --avail-repositories."),
                                'strlist', 'store', self.default_repositorypath),
+            'software-installdir-naming-scheme': ("Naming scheme for software install (sub)directory "
```

Comment:
worst name ever? why is this different btw? can this be handled in the regular naming scheme?